src/components/FilterPanel/FilterPanelAccordion/__tests__/FilterPanelAccordion.spec.js

```diff
@@ -9,19 +9,29 @@ import { render } from '@testing-library/react';
 import FilterPanelAccordion from '../FilterPanelAccordion';
 
 describe('FilterPanelAccordion', () => {
-  it('renders with a title', () => {
+  test('should have no Axe or DAP violations', async () => {
+    const main = document.createElement('main');
```

Comment:
Just a suggestion, it might be useful to create a [custom renderer](https://testing-library.com/docs/react-testing-library/setup#custom-render) for a11y tests that handle the creation of the main element and setting it as a container.

We do this in DE when we need to wrap our container with a Redux provider to work. I'm happy to share the example offline if you're curious.

Reply:
Good suggestion! Two reasons I'm hesitant to do that...
1. There may be situations where this isn't necessary, like if we end up having a component with its own sufficient landmarks. (Haven't come across one yet. For some reason, I imagine the `Shell` would apply here?)
2. Less of a reason, but -- this seems like a good way to make people remember the landmark requirements. 😆